### ISSUE TITLE
Investigate non-idempotency of the 'make i18n' command

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#8915]
## Observed behavior
Encountered in #8911  and #8804 (relevant [Slack conversation](https://learningequality.slack.com/archives/CB37UM23A/p1639435993381500)). 

## Errors and logs
…

## Expected behavior
…

## User-facing consequences
…

## Steps to reproduce
…

## Context
…

**Comment by radinamatic** [comment]
A bit of history 🔍 :
* During the recent string freeze I have run `make i18n-download...` several times until beginning of November without issues, adding the new locales [`ha`](https://github.com/learningequality/kolibri/pull/8352), [`id` and `ka`](https://github.com/learningequality/kolibri/pull/8408), and translations that had been added on Crowdin.
* Problems started when I tried to add the [`pt-mz`](https://github.com/learningequality/kolibri/pull/8572) locale. 
* @indirectlylit took over with some additional changes, merged #8624 and #8633, which eventually included the `pt-mz` locale and translations.
* When I opened  #8804, we had the first appearance of `pt-mz` translations disappearing on my side after running `make i18n-download...`, while when @indirectlylit run it on his, the translations were included.
* The issue reappeared when I opened #8911, at which point @rtibbles was able to replicate the `pt-mz` translations disappearing after running `make i18n-download...` (both of us have the dev environment in Ubuntu, while @indirectlylit works on macOS). 
* Finally, I'm again experiencing the same while pulling the recently approved `el` locale (Greek) from Crowdin: the new locale and translations work perfectly in my dev environment, but `pt-mz` is missing (again).

![Kolibri-speaks-Greek](https://user-images.githubusercontent.com/1457929/146318758-832c0dce-b3fc-42ab-bff2-244cf4f08391.png)

For the fast patch @indirectlylit could add the `el` locale (I opened #8939 pro-forma), which will also add the 2 pending Spanish correct translations, since `make i18n-download...` works for him and does not delete `pt-mz` translated strings, and we decided with the implementations team that the Greek translations need to be included in the 0.15, so a partner can do some linguistic QA (therefore I'm changing this issue to P0).
 
It would be best if we could figure out what the underlying problem is before @indirectlylit wraps up for the holidays (and going forward 🥲).
